.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'taqueria-plugin-*/docker/*'
+  create:
+    tags:
+      - 'v*'
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+
+  build-and-push-docker:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - plugin: taqueria-plugin-flextesa 
```

Comment:
I think it is useful to run this job as a matrix. This way we can add any new plugins that require docker builds to the matrix. I am happy to remove it for now if needed

Reply:
As discussed in the SRE standup the `flextesa` docker image is an outlier. I will remove the matrix configuration